docs/commerce/checkout/configure_checkout.md

```diff
@@ -0,0 +1,226 @@
+                        translation_domain: checkout
+```
+
+## Configure shipping and payment methods
+
+You can define the shipping and payment methods.
+Under `ibexa.repositories.<repository_name>.checkout`, create entries that resemble 
+the following example:
+
+``` yaml 
+ibexa:
+    repositories:
+        <repository_name>:
+            checkout:
+                workflow: <your_workflow_identifier> #optional, "ibexa_checkout" is used by default
+                shipping_methods:
+                    courier:
+                        name: "Courier"
+                        translation_domain: "checkout" #optional, all shipping/payment methods use this one by default
+                    parcel_machine:
+                        name: "Parcel machine"
+                    self_pickup:
+                        name: "Self pickup"
+                    express_delivery:
+                        name: "Express delivery"
+                payment_methods:
+                    credit_card:
+                        name: "Credit card"
+                    paypal:
+                        name: "Paypal"
+                    money_transfer:
+                        name: "Money transfer"
+```
+
+## Shipping and billing address field format configuration 
+
+In your implementation, you may need to create custom format configurations 
+for the shipping or billing address fields, for example, to use different address 
+formats based on the buyer's geographical location.
+
+Field formats for the billing and shipping addresses comply with the [FieldType Address](addressfield.md#formats) specification and can be controlled with the `billing_address_format` and `shipping_address_format` flags, respectively.
+They fall back to `billing` and `shipping` predefined formats by default:
+
+- `billing` is part of the `ibexa/corporate-accounts` repository 
+- `shipping` is part of the `ibexa/checkout` bundle's default configuration 
+
+To modify address formats you can either override the existing formats or create 
+custom ones.
+
+### Override existing Address Field Type formats 
+
+To override the existing `billing` and `shipping` address Field Type formats, under `ibexa.repositories.<repository_name>.checkout`, define the address format, 
+that has been modified, for example, by adding a tax identifier to the `billing` address:
+
+``` yaml 
+ibexa_field_type_address:
+    formats:
+        shipping:
+            country:
+                default:
+                    - region
+                    - locality
+                    - street
+                    - postal_code
+                    - email
+                    - phone_number
+                    
+        billing:
+            country:
+                default:
+                    - region
+                    - locality
+                    - street
+                    - postal_code
+                    - email
+                    - phone_number
+                    - tax_number
+```
```

Comment:
This sections needs to be removed. The `billing_address_format` / `shipping_address_format` configuration keys are meant to be used instead of overriding the existing formats as they may be used in other parts of the system, e.g. `billing` is the default format in Corporate Account address book.